stdlib/experimental/mqtt/to_test.go

```diff
@@ -56,6 +104,64 @@ from(bucket:"mybucket") |> mqtt.to(broker: "tcp://iot.eclipse.org:1883", timeout
 	}
 }
 
+const broker = "tcp://mqtt.eclipseprojects.io:1883" // "tcp://iot.eclipse.org:1883" seems not available anymore
```

Comment:
minor: all three const declarations can go inside const ( )

Reply:
Thank you!